test/metabase/db/connection_test.clj

```diff
@@ -16,31 +19,31 @@
   (fixtures/initialize :db))
 
 (deftest nested-transaction-test
-  (let [user-1                    (mt/random-email)
-        user-2                    (mt/random-email)
-        user-exists?              (fn [email]
-                                    (t2/exists? :model/User :email email))
-        create-user!              (fn [email]
-                                    (t2/insert! :model/User (assoc (t2.with-temp/with-temp-defaults :model/User) :email email)))
-        transaction-exception     (Exception. "(Abort the current transaction)")
+  (let [user-1 (mt/random-email)
+        user-2 (mt/random-email)
+        user-exists? (fn [email]
+                       (t2/exists? :model/User :email email))
+        create-user! (fn [email]
+                       (t2/insert! :model/User (assoc (t2.with-temp/with-temp-defaults :model/User) :email email)))
+        transaction-exception (Exception. "(Abort the current transaction)")
         is-transaction-exception? (fn is-transaction-exception? [e]
                                     (or (identical? e transaction-exception)
                                         (some-> (ex-cause e) is-transaction-exception?)))
-        do-in-transaction         (fn [thunk]
-                                    (try
-                                      (t2/with-transaction []
-                                        (thunk))
-                                      (catch Throwable e
-                                        (when-not (is-transaction-exception? e)
-                                          (throw e)))))]
+        do-in-transaction (fn [thunk]
+                            (try
+                              (t2/with-transaction []
+                                (thunk))
+                              (catch Throwable e
+                                (when-not (is-transaction-exception? e)
+                                  (throw e)))))]
     (testing "inside transaction"
       (do-in-transaction
        (fn []
          (create-user! user-1)
          (is (user-exists? user-1))
          (testing "inside nested transaction"
-           ;; do this on a separate thread to make sure we're not deadlocking, see
-           ;; https://github.com/seancorfield/next-jdbc/issues/244
+            ;; do this on a separate thread to make sure we're not deadlocking, see
```

Comment:
there's lots of noise in this diff. We should get your editor to agree with our comment indents and kick them out of this diff

Reply:
Yes, I need to get that figured in out in general. It doesn't seem to have an option to directly import the cljfmt settings from what I could tell on quick first-glance

In this case, the problem seems more in the the original code? Because I don't trust my IDE formatter, I'm running cljfmt before committing and that was what changed the formatting there. I reverted the changes for now, though.

Reply:
Still looks a bit off — comments should generally be aligned with the following form. e.g. on these lines (44-45) the original indentation was correct.

cljfmt doesn't touch comments, which be the source of some issues — it'll move the code around but you might still need to manually fix comments. https://metaboat.slack.com/archives/CKZEMT1MJ/p1731704390488579